### ISSUE TITLE
Fields named "null" are incorrectly parsed as invalid

### ISSUE THREAD

**Issue by PowerKiKi** [jimkyndemeyer/js-graphql-intellij-plugin#239]
## Version and Environment Details
__Operation system__: 

Ubuntu 18.04.2 LTS

__IDE name and version__: 

PhpStorm 2019.1
Build #PS-191.6183.95, built on March 28, 2019

__Plugin version__: 

JS GraphQL (2.0.0)

## Expected Behaviour

The following schema should be parsed without error:

```graphql
input Foo {
  null: String
}
```

As [per spec](https://graphql.github.io/graphql-spec/draft/#sec-Names) the only restriction on field names is the regexp `/[_A-Za-z][_0-9A-Za-z]*/` and thus "null", while probably not very common, is a valid field name.

## Actual Behaviour

The following error is shown and it will break (in a non-obvious way) all other types depending on this one:

```
Invalid Syntax : Unexpected token: "null"
```

![image](https://user-images.githubusercontent.com/72603/55855106-ad3cd300-5bba-11e9-9610-885ac36a4101.png)

## Steps to Reproduce / Link to Repo with Reproduction and Instructions

Copy/paste the schema above in probably any `.graphql` file.

I suspect the root of this issue actually is in another project. If you can confirm this, I'll gladly open a new issue where it would be the most appropriate.

**Comment by Markademus** [comment]
Fields named "on" show the same behavior. Using version 2.1.0

**Comment by shakyShane** [comment]
This is causing a big issue for the Magento community since it has a field called 'null' on an input

**Comment by PowerKiKi** [comment]
[graphql-java 14.0](https://github.com/graphql-java/graphql-java/releases/tag/v14.0) was released a few days ago. Updating to that new version should automatically fix this breaking issue.

Would it be possible increase the priority of this bug ?

**Comment by jimkyndemeyer** [comment]
Thanks for the heads up.

There's a number of breaking API changes when upgrading from graphql-java 12 to 14, some of which produce compile errors in the plugin. These also need to be addressed.

I can't say when a fix will land, but I do see this as one of the next issues that need to be fixed.

Best regards,
Jim.

**Comment by jimkyndemeyer** [comment]
Fixed in the upcoming 2.4.0.
